source/blog/042-wide-gamut-colors-in-sass.md

```diff
@@ -0,0 +1,279 @@
+
+In the same `color()` function, we can also access the 'device independent' (and gamut-less) `xyz` color spaces -- often used as an international baseline for converting between different color models. I won't get into [white points](https://www.w3.org/TR/css-color-4/#white-point) here, but we can specify `xyz-d65` (the default) explicitly, or use `xyz-d50` instead.
+
+Working out from `xyz`, we get a number of new *theoretically unbounded* color formats -- prioritizing *perceptually uniform* distribution over clean geometry. These are available in functions of their own, including `lab()` (`lightness`, `a`, and `b`) and `lch()` (`lightness`, `chroma`, and `hue`) along with the newer 'ok' versions of each -- `oklab()` and `oklch()`. If you want the full history of these formats, [Eric Portis has written a great explainer](https://ericportis.com/posts/2024/okay-color-spaces/).
+
+## TL;DR top priority new formats
+
+For the color experts, it's great to have all this flexibility. For the rest of us, there are a few stand-out formats:
+
+- `color(display-p3 …)` provides access to a wider gamut of colors, which are available on many modern displays, while maintaining a clear set of gamut boundaries.
+- `oklch(…)` is the most intuitive and perceptually uniform space to work in, a newer alternative to `hsl(…)` -- `chroma` is very similar to `saturation`. But there are no guard rails here, and it's easy to end up outside the gamuts that any screen can display, or even outside the realm of physical reality.
+- For transitions and gradients, if we want to go directly between hues (instead of going around the color wheel), `oklab(…)` is a good linear option. Usually, a transition or gradient between two in-gamut colors will stay in gamut -- but we can't always rely on that when we're dealing with extremes of saturation or lightness.
+
+## CSS color functions in Sass
+
+Sass now accepts all the new CSS formats, and treats them as first-class *colors* that we can manipulate, mix, convert, and inspect. These functions are all available globally:
+
+- `lab()`, `oklab()`, `lch()`, and `oklch()`
+-  `color()` using the `sRGB`, `srgb-linear`, `display-p3`, `a98-rgb`, `prophoto-rgb`, `rec2020`, `xyz`, `xyz-d65`, and `xyz-d50` color spaces
+- `hwb()` (Sass previously had a `color.hwb()` function, which is now deprecated in favor of the global function)
+
+The Sass color functions use the same syntax as the CSS functions, which means that a given color can be represented in a variety of different spaces. For example, these are all the same color:
+
+```scss
+$named: MediumVioletRed;
+$hex: #C71585;
+$hsl: hsl(322.2 80.91% 43.14%);
+$oklch: oklch(55.34% 0.2217 349.7);
+$display-p3: color(display-p3 0.716 0.1763 0.5105);
+```
```

Comment:
Make this a proper example, maybe with `@debug`s so that it'll do something visible in the playground:

```suggestion
{% codeExample 'color-fns', false %}
  @debug MediumVioletRed;
  @debug #C71585;
  @debug hsl(322.2 80.91% 43.14%);
  @debug oklch(55.34% 0.2217 349.7);
  @debug color(display-p3 0.716 0.1763 0.5105);
  ===
  @debug MediumVioletRed
  @debug #C71585
  @debug hsl(322.2 80.91% 43.14%)
  @debug oklch(55.34% 0.2217 349.7)
  @debug color(display-p3 0.716 0.1763 0.5105)
{% endcodeExample %}
```

Same below.